src/Application.php

```diff
@@ -35,10 +38,33 @@ class Application extends BaseApplication
      */
     public function bootstrap(): void
     {
+        // Call parent to load bootstrap from files.
         parent::bootstrap();
 
+        if (PHP_SAPI === 'cli') {
+            $this->bootstrapCli();
+        }
+
+        /*
+         * Only try to load DebugKit in development mode
+         * Debug Kit should not be installed on a production system
+         */
+        if (Configure::read('debug')) {
+            $this->addPlugin('DebugKit');
+        }
+
         // Load 'BEdita/WebTools' and other plugins here
-        $this->addPlugin('BEdita/WebTools', ['bootstrap' => true]);
+        $this->addPlugin('BEdita/WebTools');
+        $this->addPlugin('BEdita/I18n');
```

Comment:
I would add a commented line like 
```// To activate I18nMiddleware use $this->addPlugin('BEdita/I18n', ['middleware' => true]);```
to remind the `middleware` options

Reply:
Yes it's better. I'm going to add it.